website/source/docs/job-specification/group.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "docs"
+page_title: "group Stanza - Job Specification"
+sidebar_current: "docs-job-specification-group"
+description: |-
+  The "group" stanza defines a series of tasks that should be co-located on the
+  same Nomad client. Any task within a group will be placed on the same client.
+---
+
+# `group` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> **group**</code>
+    </td>
+  </tr>
+</table>
+
+The `group` stanza defines a series of tasks that should be co-located on the
+same Nomad client. Any [task][] within a group will be placed on the same
+client.
+
+```hcl
+job "docs" {
+  group "example" {
+    # ...
+  }
+}
+```
+
+## `group` Parameters
+
+- `count` `(int: 1)` - Specifies the number of the task groups that should
+  be running under this group. This value must be non-negative.
+
+- `constraint` <code>([Constraint][]: nil)</code> -
+  This can be provided multiple times to define additional constraints.
+
+- `restart` <code>([Restart][]: nil)</code> - Specifies the restart policy for
+  all tasks in this group. If omitted, a default policy exists for each job
+  type.
+
+- `task` <code>([Task][]: required)</code> - Specifies one or more tasks to run
+  within this group. This can be specified multiple times, to add a task as part
+  of the group.
+
+- `meta` <code>([Meta][]: nil)</code> - Specifies a key-value map that annotates
+  with user-defined metadata.
+
+## `group` Examples
+
+The following examples only show the `group` stanzas. Remember that the `group`
+Remember that the `group` stanza is only valid in the placements listed above.
+
+### Specifying Count
+
+This example specifies that 5 instances of the tasks within this group should be
+running:
+
+```hcl
+group "example" {
+  count = 5
+}
+```
+
+### Task with Constraint
```

Comment:
Tasks with Constraint? Since if it is just one task we could put the constraint on the task. Could see it being confusing to people.

If we say we can factor the common constraint and place it on the group that makes sense